### PR TITLE
Add profile fragment

### PR DESCRIPTION
**Pull request by jeamigue**
This commit mainly add the Profile fragment. It is a good example of how to add a fragment to the project.
Other minor changes have been made, such as fixing the changeFragment of the MainActivity (now working), or making the authentication imageView clickable.

### CODE REVIEW

File: app/src/main/java/com/github/bjolidon/bootcamp/MainActivity.kt
Comment (JulienDiTria) [inline app/src/main/java/com/github/bjolidon/bootcamp/MainActivity.kt:1]: might need a couple more comments on the onCreate to understand what is what

File: app/src/main/java/com/github/bjolidon/bootcamp/ui/profile/ProfileFragment.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      textView.text = it
+    }
+    return root
+  }
+
+override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
+}
```
Comment (JulienDiTria) [inline app/src/main/java/com/github/bjolidon/bootcamp/ui/profile/ProfileFragment.kt:41]: missing new line at end of file

File: app/src/main/java/com/github/bjolidon/bootcamp/ui/profile/ProfileViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class ProfileViewModel : ViewModel() {
+
+    private val _text = MutableLiveData<String>().apply {
+        value = "Coming soon here: a profile editor"
+    }
+    val text: LiveData<String> = _text
+}
```
Comment (JulienDiTria) [inline app/src/main/java/com/github/bjolidon/bootcamp/ui/profile/ProfileViewModel.kt:13]: end of file new line

File: app/src/main/res/layout/activity_authentication.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
     android:layout_height="match_parent">
 
     <ImageView
-        android:id="@+id/imageView"
+        android:id="@+id/profileIcone"
```
Comment (JulienDiTria) [inline app/src/main/res/layout/activity_authentication.xml:10]: why has this been changed here ? seems like nothing to do with your profileFragment

**Review by JulienDiTria — CHANGES_REQUESTED** [review]

**Comment by JulienDiTria** [comment]
There is a weird behavior : when clicking the demo button for frag switch, it goes on browse as expected, but then the clicking on home on the navigation doesn t work anymore.
Is there a ranking of navigation etc ?

File: app/src/main/res/layout/activity_authentication.xml (hunk 3 above)
Comment (jeamigue) [reply app/src/main/res/layout/activity_authentication.xml:10]: It does though: it's on this icon you have to click to launch the profileFragment.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -15,7 +15,7 @@
     <string name="nav_header_title">Android Studio</string>
     <string name="nav_header_subtitle">android.studio@android.com</string>
     <string name="nav_header_desc">Navigation header</string>
-    <string name="action_settings">Settings</string>
+    <string name="action_settings">Settlings</string>
```
[the remark this answers is not in the document]
Comment (jeamigue) [reply app/src/main/res/values/strings.xml:18]: accident. :p

**Comment by Bjolidon** [comment]
Seems good to me

**Review by JulienDiTria — APPROVED** [review]
good job

**Pull request merged**